specification/devcenter/DevCenter/Environments/routes.tsp

```diff
@@ -169,6 +169,11 @@ interface Environments {
       @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
       @path
       environmentName: string;
+
```

Comment:
Again, as in my review meeting comment and comment in PR, please move these big blobs of query parameter into a separate file say `QueryParams.tsp` and spread it here.

Reply:
moved to parameters.tsp file. Also moved other repeated path parameters to that file